commit 3eb09a3e028e26fd26bc132c7aa0577f00de2d05
Author: Oliver Upton <oliver.upton@linux.dev>
Date:   Fri Oct 25 20:31:06 2024 +0000

    KVM: arm64: selftests: Add tests for MMIO external abort injection
    
    Test that the plumbing exposed to userspace for injecting aborts in
    response to unexpected MMIO works as intended in two different flavors:
    
     - A 'normal' MMIO instruction (i.e. ESR_ELx.ISV=1)
    
     - An ISV=0 MMIO instruction with/without KVM_CAP_ARM_NISV_TO_USER
       enabled
    
    Reviewed-by: Marc Zyngier <maz@kernel.org>
    Link: https://lore.kernel.org/r/20241025203106.3529261-5-oliver.upton@linux.dev
    Signed-off-by: Oliver Upton <oliver.upton@linux.dev>

diff --git a/tools/testing/selftests/kvm/Makefile b/tools/testing/selftests/kvm/Makefile
index 960cf6a77198..98957a99ead6 100644
--- a/tools/testing/selftests/kvm/Makefile
+++ b/tools/testing/selftests/kvm/Makefile
@@ -156,6 +156,7 @@ TEST_GEN_PROGS_aarch64 += aarch64/aarch32_id_regs
 TEST_GEN_PROGS_aarch64 += aarch64/arch_timer_edge_cases
 TEST_GEN_PROGS_aarch64 += aarch64/debug-exceptions
 TEST_GEN_PROGS_aarch64 += aarch64/hypercalls
+TEST_GEN_PROGS_aarch64 += aarch64/mmio_abort
 TEST_GEN_PROGS_aarch64 += aarch64/page_fault_test
 TEST_GEN_PROGS_aarch64 += aarch64/psci_test
 TEST_GEN_PROGS_aarch64 += aarch64/set_id_regs